spec/features/system/admin/instance_management_spec.rb

```diff
@@ -48,8 +48,10 @@
       end
 
       scenario 'I can see all instances' do
-        instances = create_list(:instance, 2)
-        visit admin_instances_path(page: Instance.page.total_pages)
+        create_list(:instance, 2)
+        last_page = Instance.page.total_pages
+        instances = Instance.order_by_id.page(last_page)
```

Comment:
@lowjoel I noticed that there is an `order_by_id` in the controller, so I also added it here. But why we need it ? I think the default order is by id?